ui/src/app/integrations/list/list.component.scss

```diff
@@ -10,6 +11,7 @@
       }
     }
   }
+  */
```

Comment:
If this piece of CSS is no longer required, I'd say we should remove it instead of commenting it out.

Reply:
Yup, totally meant to do that, will take it out.